composables/settings/definition.ts

```diff
@@ -15,7 +15,7 @@ export interface PreferencesSettings {
   hideFollowerCount: boolean
   hideTranslation: boolean
   hideUsernameEmojis: boolean
-  hideAccountHoverCard: boolean
+  hideAccountAndTagHoverCard: boolean
```

Comment:
Let's split this in two, we keep `hideAccountHoverCard` and we add a new `hideTagHoverCard`. I think it is better so we respect current preferences and also I imagine there are folks that will care about the accounts but would prefer to hide the tags.

Reply:
Yes, that makes sense as they had a bit different roles. I added `hideTagHoverCard` as a separate one.